### DISCUSSION TITLE
Adobe Flash Player Problem

### DISCUSSION THREAD

**Discussion by cheriathesummonrr** [lightspark/lightspark#640] [General]
Will it help with Adobe Flash Player so i can play it again

**Comment by dbluelle** [comment]
you can use lightspark as a replacement for the adobe flash player, but you need to use an alternative browser, because chrome and firefox stopped supporting the flash plugin.

**Comment by cheriathesummonrr** [comment]
What kind of alternative browser I should be using

**Reply by huwdp** [reply]
Waterfox and Palemoon are two that I know have kept plugins
